src/main/kotlin/me/lensvol/blackconnect/BlackdClient.kt

```diff
@@ -71,6 +71,7 @@ class BlackdClient(hostname: String, port: Int, useSsl: Boolean = false) {
         fastMode: Boolean = false,
         skipStringNormalization: Boolean = false,
         skipMagicTrailingComma: Boolean = false,
+        preview: Boolean = false,
```

Comment:
Here and everywhere else it's better to label it more explicitly: `preview` -> `previewMode`.

Reply:
Thank you for doing this! But I feel that we need to account for the fact that this feature is only available on the specific **blackd** versions. And while a lot of people are using bleeding edge versions of **black**, a lot of them also have older pinned versions in their projects. Especially in corporate environments as I know from experience.

You need to check if the specific `blackd` version used during the call supports that particular feature and show an error to the user if server's version is too low. Otherwise they may not get the expected result.